### PR TITLE
Add tests for github actions problem matchers

### PR DESCRIPTION
**Pull request by btkostner**
I figured I would setup testing for the problem matchers before I start adding some. As requested in https://github.com/erlef/setup-beam/issues/62#issuecomment-886267969

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Oh, you did it for the existing stuff. That's nice!

**Comment by paulo-ferraz-oliveira** [comment]
To avoid waiting for the action to throw due to static analysis or formatting issues, you can run `npm run build && npm run format && npm run jslint` while you go.

**Comment by btkostner** [comment]
Missed the `jslint` :sweat_smile:

**Comment by btkostner** [comment]
Seems GitHub is having issues with it's runners. 1 failed (not because of changes, but some internal GitHub error), and 3 are still in queue after 20ish hours.

**Comment by paulo-ferraz-oliveira** [comment]
I'm re-running all the jobs.

**Comment by paulo-ferraz-oliveira** [comment]
Also, given this first set of tests is working (as proven before) would you be Ok to merge this and start a new PR for `main`, as soon as the actions succeed? This would avoid the need to approve your workflows as a first-time contributor.

**Comment by paulo-ferraz-oliveira** [comment]
All checks have passed 😄

File: __tests__/problem-matchers.test.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ async function testElixirMixCompileWarning() {
   ).pattern
 
   const firstOutput =
-    'warning: variable "err" does not exist and is being expanded to "err()", please use parentheses to remove the ambiguity or change the variable name'
```
Comment (paulo-ferraz-oliveira) [inline __tests__/problem-matchers.test.js]: Not really important, but `npm format` should do this for you. Doesn't it?

**Comment by btkostner** [comment]
I would be fine with merging this now and opening another PR for adding some more problem matchers

File: __tests__/problem-matchers.test.js (hunk 1 above)
Comment (btkostner) [reply __tests__/problem-matchers.test.js]: `npm format` won't break up a line if it's too long

Comment (paulo-ferraz-oliveira) [reply __tests__/problem-matchers.test.js]: Fun stuff!

**Review by paulo-ferraz-oliveira — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Squash-merging to `master`, to allow for a further pull request where we avoid the need to constantly keep approving CI runs.

**Pull request merged**